packages/ChangesReloaded-Core.package/CRBrowser.class/instance/buildChangeListSpecWith..st

```diff
@@ -3,8 +3,9 @@ buildChangeListSpecWith: aBuilder
 
 	^ aBuilder pluggableListSpec new
 		model: self;
-		name: self class changeListName;
+		name: CRBrowser changeListName;
```

Comment:
No, why? Afaik `self class` is the better option... See [slide deck SWA U08 Idiom 116. Avoid using explicit class names](https://hpi.de/internalfiles/materialien/FG%20Software-Architekturen/SWA1920_V/Handouts/Swa_U08_LanguageIdioms_SmalltalkThomas.pdf#page=61)
```suggestion
		name: self class changeListName;
```